internal/appliance/golden_test.go

```diff
@@ -0,0 +1,130 @@
+package appliance
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+)
+
+// Test helpers
+
+// creationTimestamp and uid need to be normalized
+var magicTime = metav1.NewTime(time.Date(2024, time.April, 19, 0, 0, 0, 0, time.UTC))
+
+type goldenFile struct {
+	Resources []client.Object `json:"resources"`
+}
+
+func (suite *ApplianceTestSuite) makeGoldenAssertions(namespace, goldenFileName string) {
```

Comment:
```suggestion
func (s *ApplianceTestSuite) makeGoldenAssertions(namespace, goldenFileName string) {
```

Nit: but the receiver name is different than in `kubernetes_test.go`

Reply:
In that file, it would shadow the package name. That's probably fine though, tbh